docs/interaction/rpc.md

```diff
@@ -10,118 +10,508 @@ The following methods are available:
+        "outgoing_receipts_root": "6m2bJP9TiEtxqHSzLygPCQqiGdgYQQYaSYTFeP2gEUQj",
+        "prev_block_hash": "7XkFdjUSXXCGdgoZbpdM5DehXSh7uAu7eXeMdVBxRU2L",
+        "prev_state_root": "582a9JD8Ts8vMKqJSsLRmvdEm9FyLYrx1Ssujc5hSkL9",
+        "rent_paid": "0",
+        "shard_id": 0,
+        "signature": "ed25519:3rQsYzWqRfAhqADMcH2JLPPsubHcBp3yBJNeHexAZDGrStb72rNb6vL2JctZkqyFwHxmLvaEMwaowtthmmu3Y64k",
+        "tx_root": "11111111111111111111111111111111",
+        "validator_proposals": [],
+        "validator_reward": "0"
+      },
+      /* ... continued for each chunk ... */
+    ],
+    "header": {
+      "approvals": [
+        [
+          "far",
+          "7XkFdjUSXXCGdgoZbpdM5DehXSh7uAu7eXeMdVBxRU2L",
+          "3rURN1sFxUwQVzWpnPyrMjxZftd4QMj54RTDs1Lin3MC",
+          "ed25519:5dxfFjqaJsWVPuH9gXWJtYdeMpBVcKcYfbH8RPFGvvX6vq9h3uv4CrwvmaDUgXv2wmLEsEANPGq4b2KJR5uiuuVk"
+        ]
+      ],
+      "challenges_result": [],
+      "challenges_root": "11111111111111111111111111111111",
+      "chunk_headers_root": "9zcPf9kndKeHuNptPn6KPamaW3AfikmcrVp57uH4Y6x5",
+      "chunk_mask": [
+        true,
+        true,
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
+      ],
+      "chunk_receipts_root": "CNktChEct984axKmREForA86DmHfRRaKr55urUqA9Nfq",
+      "chunk_tx_root": "7ADhq2WENoYFqd7SPga7n8hS1juBNJJJgJYoy7czSaEc",
+      "chunks_included": 8,
+      "epoch_id": "3rURN1sFxUwQVzWpnPyrMjxZftd4QMj54RTDs1Lin3MC",
+      "gas_price": "100",
+      "hash": "BQLidbFoQf62MQBGURokMEy6U1pSvNWio3ANBKKVU5hu",
+      "height": 1000,
+      "last_quorum_pre_commit": "B58SjfZ86mrw61Rcvg76uZNQZCdfPudzyYmCRF15ehFz",
+      "last_quorum_pre_vote": "7XkFdjUSXXCGdgoZbpdM5DehXSh7uAu7eXeMdVBxRU2L",
+      "next_bp_hash": "HnXfTqBzxQzYEJptuQBnjRfJ8WpkdNPKFVfJDc7A2SUx",
+      "next_epoch_id": "79mKZobxgrugozxmcu5399QFwSsqHBPzaDziLAWERtyq",
+      "outcome_root": "7ADhq2WENoYFqd7SPga7n8hS1juBNJJJgJYoy7czSaEc",
+      "prev_hash": "7XkFdjUSXXCGdgoZbpdM5DehXSh7uAu7eXeMdVBxRU2L",
+      "prev_state_root": "FDCj1SaJ9YaAMVYfoPuFNTS17bP4eY5vXtH1qZzQFqvt",
+      "rent_paid": "0",
+      "score": "19591494011875585000000000",
+      "signature": "ed25519:3puQQENzDsPgg6oV89f5QH7U84ziMrDFKxv3xNSfdsZrAmTTPee2KWWsE39Rjd2KWHpGHEyJ76gkHjMw5THKuBXJ",
+      "timestamp": 1579220301061391600,
+      "total_supply": "1638150454593810427394246",
+      "total_weight": "19591495037209980000000000",
+      "validator_proposals": [],
+      "validator_reward": "0"
+    }
+  },
+  "id": "dontcare"
+}
+```
+
+## Transaction Status Endpoint
+
+`tx(txHash: string)`
+
+Queries status of a transaction by hash and returns `FinalTransactionResult`.
+
+- HTTP `POST`
+- URL `https://rpc.nearprotocol.com`
+- method `tx`
+- params
+  - (1) `<base 58 of transaction hash>` (see [NEAR Explorer](https://explorer.nearprotocol.com) for a valid transaction hash)
+  - (2) `<sender account id>` (ignored)
```

Comment:
required -- is to determine which shard to look for transaction

Reply:
✅